scripts/yggdrasil-iptunnel/uninstall

```diff
@@ -4,10 +4,4 @@ set -e
 
 # Uninstall scripts that configure yggdrasil iptunnel
 sudo rm /usr/local/sbin/yggdrasil-setup
-sudo rm /usr/local/sbin/yggdrasil-pre-setup
-if [ -f /lib/systemd/system/yggdrasil.service ]; then
-    sudo sed -i /ExecStartPost/d /lib/systemd/system/yggdrasil.service
-fi
-if [ -f /etc/systemd/system/yggdrasil.service ]; then
-    sudo sed -i /ExecStartPost/d /etc/systemd/system/yggdrasil.service
-fi
+sudo rm /etc/systemd/system/yggdrasil.service.d/iptunnel.conf
```

Comment:
Maybe ask about removing this one? I would also ask and remove the other config files, server and client ones.

Reply:
I would also remove the service here, and then do a `sudo systemctl daemon-reload`.

Reply:
this removes the iptunnel augmentation to the Yggdrasil service ONLY
leaves yggdrasill intact